hw/ip/csrng/doc/_index.md

```diff
@@ -91,3 +91,202 @@ This document aims to make the distinction between physical entropy and factory
+{signal: [
+   {name: 'clk'            , wave: 'p...............|.....'},
+   {name: 'csrng_req_vld'  , wave: '01............0.|.....'},
+   {name: 'csrng_req_rdy'  , wave: '1.............0.|..1..'},
+   {name: 'csrng_req_bus'  , wave: 'x5333333333333x.|.....',data: ['ins','sd1','sd2','sd3','sd4','sd5','sd6','sd7','sd8','sd9','sd10','sd11','sd12']},
+   {name: 'csrng_rsp_ack'  , wave: '0...............|.10..'},
+   {name: 'csrng_rsp_sts'  , wave: 'x...............|.5x..', data: ['ok']},
+ {},
+]}{{< /wavejson >}}
+
+##### Reseed Request
+
+{{< wavejson >}}
+{signal: [
+   {name: 'clk'            , wave: 'p...............|.....'},
+   {name: 'csrng_req_vld'  , wave: '01............0.|.....'},
+   {name: 'csrng_req_rdy'  , wave: '1.............0.|..1..'},
+   {name: 'csrng_req_bus'  , wave: 'x5333333333333x.|.....',data: ['res','ad1','ad2','ad3','ad4','ad5','ad6','ad7','ad8','ad9','ad10','ad11','ad12']},
+   {name: 'csrng_rsp_ack'  , wave: '0...............|.10..'},
+   {name: 'csrng_rsp_sts'  , wave: 'x...............|.5x..', data: ['ok']},
+ {},
+]}{{< /wavejson >}}
+
+##### Generate Request
+
+{{< wavejson >}}
+{signal: [
+   {name: 'clk'            , wave: 'p......|..............'},
+   {name: 'csrng_req_vld'  , wave: '01....0|..............'},
+   {name: 'csrng_req_rdy'  , wave: '1.....0|.............1'},
+   {name: 'csrng_req_bus'  , wave: 'x53333x|..............',data: ['gen','ad1','ad2','ad3','ad4']},
+   {name: 'csrng_rsp_ack'  , wave: '0......|............10'},
+   {name: 'csrng_rsp_sts'  , wave: 'x......|............5x', data: ['ok']},
+   {name: 'genbits_vld'    , wave: '0......|.1...01......0'},
+   {name: 'genbits_bus'    , wave: 'x......|.4444x4x44444x', data: ['bits','bits','bits','bits','bits','bits','bits','bits','bits','bits']},
+   {name: 'genbits_rdy'    , wave: '1......|.......01.....'},
+ {}, {},
+]}{{< /wavejson >}}
+
+##### Update Request
+
+{{< wavejson >}}
+{signal: [
+   {name: 'clk'            , wave: 'p...............|.....'},
+   {name: 'csrng_req_vld'  , wave: '01............0.|.....'},
+   {name: 'csrng_req_rdy'  , wave: '1.............0.|..1..'},
+   {name: 'csrng_req_bus'  , wave: 'x5333333333333x.|.....',data: ['upd','ad1','ad2','ad3','ad4','ad5','ad6','ad7','ad8','ad9','ad10','ad11','ad12']},
+   {name: 'csrng_rsp_ack'  , wave: '0...............|.10..'},
+   {name: 'csrng_rsp_sts'  , wave: 'x...............|.5x..', data: ['ok']},
+ {},
+]}{{< /wavejson >}}
+
+##### Uninstantiate Request
+
+{{< wavejson >}}
+{signal: [
+   {name: 'clk'            , wave: 'p...............|.....'},
+   {name: 'csrng_req_vld'  , wave: '010.............|.....'},
+   {name: 'csrng_req_rdy'  , wave: '1.0.............|..1..'},
+   {name: 'csrng_req_bus'  , wave: 'x5x.............|.....',data: ['uni']},
+   {name: 'csrng_rsp_ack'  , wave: '0...............|.10..'},
+   {name: 'csrng_rsp_sts'  , wave: 'x...............|.5x..', data: ['ok']},
+ {},
+]}{{< /wavejson >}}
+
+
+
+
+
+
+### Interrupts
+
+The CSRNG module has 2 interrupts:
```

Comment:
I think you need to define these 2 interrupts here.

Reply:
More of a placeholder, but the currently defined interests have now been added.

@cdgori thanks for the review!